apps/snmptrapd_handlers.c

```diff
@@ -810,9 +810,10 @@ int   command_handler( netsnmp_pdu           *pdu,
     size_t          r_len = 64, o_len = 0;
     int             oldquick;
 
+
+    if (handler && handler->token && *handler->token) {
```

Comment:
Shouldn't this if-statement be swapped with the debug statement below it? Please also fix the indentation.

Reply:
Debug has been replaced for avoiding dereference of handler->token that can be NULL

Reply:
handler can not be NULL.  The calling sequence is

```
                ret = (*(traph->handler))(pdu, transport, traph);
```

The `handler` arg is the `traph` variable here.  If `traph` is NULL, this code will fail without calling the handler.  Therefore, the right fix is to instead remove the `handler &&` check, to avoid the appearance that it's possible for handler to be NULL.

Reply:
The above suggestion has been implemented in commit dc4ef4a5aed66f2e2048657c2c95413b3f294035. Does everyone agree that that commit is sufficient to close this pull request?